src/app/branded/step-1/branded-checkout-step-1.component.js

```diff
@@ -166,6 +177,7 @@ export default angular
       donorDetails: '<',
       defaultPaymentType: '<',
       hidePaymentTypeOptions: '<',
+      showCoverFees: '<',
```

Comment:
Yes, this is an Angular thing. Basically, it tells the component (in the HTML) that `show-cover-fees` is a valid attribute on the component and that it can accept expressions as the value of the attribute. [Here is a blog](https://blog.krawaller.se/posts/dissecting-bindings-in-angularjs/) related to it.